xml_converter/generators/cpp_templates/compoundvalue.cpp

```diff
@@ -27,18 +27,19 @@ using namespace std;
     }
     return {{attribute_name}};
 }
-{% if attribute_variables[0].xml_export == "Parent" %}
-    string stringify_{{attribute_name}}({{class_name}} attribute_value) {
-        string output;
-        {% for n, attribute_variable in enumerate(attribute_variables) %}
-            {% if n == 0: %}
-                output = to_string(attribute_value.{{attribute_variables[n].attribute_name}});
-            {% else %}
-                output = output + "," + to_string(attribute_value.{{attribute_variables[n].attribute_name}});
-            {% endif %}
-        {% endfor %}
-        return output;
-    }
+{% if attribute_variables[0].xml_parent_export != [] %}
```

Comment:
This seems strange that we are accessing the first element of the array to get the info.

Reply:
As discussed this is still weird but it should be a different pr.

Reply:
When getting the other check fixed, this ended up being a simple addition. I still think the function in code generator needs some attention though.